modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -358,14 +358,36 @@ function dump_postgresql {
 }
 
 function output_restore_sql {
-  pg_restore -j 2 "${dumpfile}" | sed -r "${sed_cmds}"
+
+  PG_RESTORE_VERSION='13.4'
```

Comment:
Rather than set the version here and have it immediately overwritten, I'd just make an empty variable:

```suggestion
  PG_RESTORE_VERSION='' # this gets set in the case statement below
```